files/build_templates/docker_image_ctl.j2

```diff
@@ -142,6 +142,14 @@ start() {
     # Obtain our HWSKU as we will mount directories with these names in each docker
     HWSKU=`sonic-netns-exec "$NET_NS" sonic-cfggen -d -v 'DEVICE_METADATA["localhost"]["hwsku"]'`
     {%- endif %}
+    {%- if docker_container_name == "snmp" %}
+    MOUNTS=`docker inspect -f '{{ .Mounts }}' {{docker_container_name}}`
+    # If previously mounted /var/run/netns exists the remove docker
```

Comment:
why?

Reply:
how do determine if it is single asic or multi-asic system?

Reply:
> why?
on the host the /var/run/netns/<> is symlink to network of database docker container process:
/var/run/nets/asic<n> --> /proc/<pid>/ns/net
If the device is rebooted or any of the database docker is restarted, /proc/pid/ns/net can be stale.
On the SNMP docker mount is done as:
(Host) /proc/<pid>/ns/net -> /var/run/nets/asic<n>
So, when systemd service is restarted, previous docker mounts might be stale (not always).

Reply:
> how do determine if it is single asic or multi-asic system?

If host has namespace files in  /var/run/netns, then it is multi-asic, only then the /var/run/netns files are mounted in SNMP docker.

Reply:
I am not sure if this is reliable to assume only multi-asic system has /var/run/netns

Reply:
you only need to access the db, why do you need to mount the namespace?

Reply:
Using netns to access arptable from all namespaces; required for https://github.com/Azure/sonic-snmpagent/pull/129

Reply:
it is better to modify snmpagent to use neighbor table in the app db so that we do not need to have this requirement.